### PR TITLE
Javascript javascript1 week1

### PR DESCRIPTION
**Pull request by sunilkumarmahajan**
Javascript week1 home work

### CODE REVIEW

File: javascript/javascript1/week1/EzNamey.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+var firstWords = ["kia", "peugeot", "vw", "toyoto", "volvo", "benz", "nissan", "skoda", "hyundai", "ferrari"];
+var secondWords = ["picanto", "benzin", "gas", "i30", "polo", "gitz", "mini", "opel", "2008", "automat"];
+var randomNumber = Math.floor(Math.random() * 10);
```
Comment (KavyaReddy732) [inline javascript/javascript1/week1/EzNamey.js:3]: you can replace 10 with length of an array (i.e *firstWords.length). this gives the value depends on array length.

File: javascript/javascript1/week1/Goodboy-Oldboy.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+var dogYearOfBirth = 2018;
+var dogYearFuture = 2025;
```
Comment (KavyaReddy732) [inline javascript/javascript1/week1/Goodboy-Oldboy.js:2]: same here as well use 'const'.

File: javascript/javascript1/week1/Goodboy-Oldboy.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+var dogYear = dogYearFuture - dogYearOfBirth;
+var shouldShowResultInDogYears = true;
+if (shouldShowResultInDogYears = true )
+{
+    console.log("Your dog will be " + (dogYear*10) +  " dog years old in " + dogYearFuture);
```
Comment (KavyaReddy732) [inline javascript/javascript1/week1/Goodboy-Oldboy.js:7]: dog age is 7 times to human age but not 10 times. this should be dogYear*7

File: javascript/javascript1/week1/Houseypricey.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+var height = 10;
+var depth = 10;
+var gardenSizeInM2 = 100;
+var actualhousePrise = 2500000;
+var volumeInMeters;
+var housePrice;
+volumeInMeters = width * height * depth;
+housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+if (housePrice > actualhousePrise) {
+console.log ("peter is paying exess price");}
```
Comment (KavyaReddy732) [inline javascript/javascript1/week1/Houseypricey.js:12]: here you can use next line after semi-colon (;) to close the brace ( } ) for neat code.

File: javascript/javascript1/week1/Houseypricey.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
+//peter house calculation
+var width = 8;
+var height = 10;
```
Comment (KavyaReddy732) [inline javascript/javascript1/week1/Houseypricey.js:3]: same here use 'const'.

File: javascript/javascript1/week1/Houseypricey.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+var actualhousePrise = 2500000;
+var volumeInMeters;
+var housePrice;
+volumeInMeters = width * height * depth;
+housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+if (housePrice > actualhousePrise) {
+console.log ("peter is paying exess price");}
+else
+{console.log("peter is paying less than the actual price");}
+//Julia house calculation
```
Comment (KavyaReddy732) [inline javascript/javascript1/week1/Houseypricey.js:15]: you can use arrays to make a simple code like this let width=[8,5];
const depth=[10,11];
const height=[10,8];
const gardenSizeInM2=[100,70];
const actualHousePrice=[2500000 , 1000000];
const VolumeInMeters=[width[0]*deep[0]*height[0] , width[1]*deep[1]*height[1]]
const housePrice = 
[VolumeInMeters[0] * 2.5 * 1000 + gardenSizeInM2[0] * 300,
     VolumeInMeters[1] * 2.5 * 1000 + gardenSizeInM2[1] * 300];
then you can compare using their index values like this :
if(actualHousePrice[0]<=housePrice[0])

**Review by KavyaReddy732 — COMMENTED** [review]
Nice understanding of code. but never use Var and use next line for closing braces at the end of statement.

File: javascript/javascript1/week1/Age-ify.js [hunk 7]
```diff
@@ -1,5 +1,4 @@
-var yearOfBirth = 1987;
-var yearFuture = 2045 ;
-var age = yearFuture - yearOfBirth;
-//console.log (age);
-console.log("You will be "+ age + " years old in " + yearFuture);
\ No newline at end of file
+const yearOfBirth = 1987;
+const yearFuture = 2045 ;
+const age = yearFuture - yearOfBirth;
+console.log(`Your will be ${age} years old in ${yearFuture}`);
```
Comment (zuhairtaha) [inline javascript/javascript1/week1/Age-ify.js:4]: Good job!

**Pull request merged**